### PR TITLE
babel: l3roamd initscript - define mesh-interfaces dynamically

### PR DESCRIPTION
**Pull request by christf**

### CODE REVIEW

File: package/gluon-l3roamd/files/etc/init.d/gluon-l3roamd [hunk 1]
```diff
@@ -9,7 +9,8 @@ start_service () {
         procd_open_instance
         procd_set_param stdout 1
         procd_set_param stderr 1
-        procd_set_param command "$PROG" -p $(lua -e 'print(require("gluon.site_config").prefix6)') -i br-client -m mesh0 -t 11 -a $(uci get network.loopback.ip6addr | cut -d/ -f1)
+	meshinterfaces=$(lua -e 'mesh = require("gluon.mesh");for interface, _ in pairs(mesh.interfaces())  do print(" -m " .. interface);end')
```
Comment (tcatm) [inline package/gluon-l3roamd/files/etc/init.d/gluon-l3roamd]: Please use spaces to indent this (like the surrounding code does).

**Comment by NeoRaider** [comment]
It would be nice if you could prefix your babel PRs with *[babel]* or *babel:* so it is clear at a glance that these are for the babel branch.

**Comment by christf** [comment]
done.

**Comment by christf** [comment]
what else needs to be done to get this merged?

**Comment by tcatm** [comment]
This won't be necessary with mmfd.

**Pull request closed without merging**